redaxo/src/addons/structure/plugins/history/lib/history_login.php

```diff
@@ -31,11 +31,17 @@ public function checkTempSession($historyLogin, $historySession, $historyValidti
         return null;
     }
 
+    /**
+     * @return null|string
```

Comment:
```suggestion
     * @return string
```

Wobei ich mir hier unsicher bin.
Laut Doku kann password_hash auch false liefern (aber nicht null). Theoretisch wäre also `false|string` richtig, aber eigentlich sehe ich unsere Methode als `string` an, als andere ist ja ein Fehler.
Oder anders gesagt: Mit echtem Returntype würde ich `string` notieren und es somit in einen Fehler laufen lassen, wenn `false` zurückkommt.